src/ebonite/ext/docker/builder.py

```diff
@@ -11,7 +10,7 @@ class DockerBuilder(BuilderBase):
     """Builder implementation to build docker images"""
     @validate_kwargs
     def create_image(self, name: str, environment: DockerEnv, tag: str = 'latest', repository: str = None,
-                     **kwargs) -> Image.Params:
+                     **kwargs) -> DockerImage:
```

Comment:
Wouldn't it cause IDE warnings about typing mismatch?

Reply:
no